File: app/src/main/java/com/veyndan/paper/reddit/deeplink/WebDeepLink.java
package com.veyndan.paper.reddit.deeplink;

import com.airbnb.deeplinkdispatch.DeepLinkSpec;

@DeepLinkSpec(prefix = {
        "http://www.reddit.com",
        "http://reddit.com",
        "https://www.reddit.com",
        "https://reddit.com"
})
public @interface WebDeepLink {
    String[] value();
}

File: app/src/main/java/com/veyndan/paper/reddit/deeplink/WebDeepLink.kt
package com.veyndan.paper.reddit.deeplink

import com.airbnb.deeplinkdispatch.DeepLinkSpec

@DeepLinkSpec(prefix = arrayOf(
        "http://www.reddit.com",
        "http://reddit.com",
        "https://www.reddit.com",
        "https://reddit.com")
)
annotation class WebDeepLink(vararg val value: String)
